Samples/NetworkMediaStreamer/CameraRTPStreamerApp/CameraRTPStreamerApp.cpp

```diff
@@ -40,7 +40,7 @@ constexpr uint16_t ServerPort = 8554;
 constexpr uint16_t SecureServerPort = 6554;
 
 // Uncomment the following if you want to use FrameReader API instead of the Record-to-sink APIs
-#define USE_FR 
+//#define USE_FR 

```

Comment:
>USE_FR [](start = 10, length = 6)

Any reason why we aren't using USE_FR path? If there are known pros and cons, perhaps we should mention here.

Reply:
There aren't any pro/cons. its just the application design. 
for e.g. ROS uses SourceReader and not Mediacapture so it has to use a SinkWriter. The use FR is only to test that path. OR another example is theoretically someone wants to create their own sample or modify/process the samples in the application before streaming out.

---
In reply to: [497845460](https://github.com/microsoft/Windows-Camera/pull/14#discussion_r497845460) [](ancestors = 497845460)

Reply:
You are changing the default path in this app as part of this change, so I am curious as to why.

---
In reply to: [497846364](https://github.com/microsoft/Windows-Camera/pull/14#discussion_r497846364) [](ancestors = 497846364,497845460)

Reply:
added comments

---
In reply to: [497846364](https://github.com/microsoft/Windows-Camera/pull/14#discussion_r497846364) [](ancestors = 497846364,497845460)